contrib/tlp/template.py

```diff
@@ -0,0 +1,43 @@
+pkgname = "tlp"
+pkgver = "1.6.1"
+pkgrel = 0
+build_style = "makefile"
+make_cmd = "gmake"
+make_install_args = [
+    "-j1",
+    "TLP_SBIN=/usr/bin",
+    "TLP_ULIB=/usr/lib/udev",
+    "TLP_ELOD=/usr/libexec/elogind/system-sleep",
+    "TLP_NO_INIT=1",
+    "TLP_WITH_ELOGIND=1",
+    "TLP_WITH_SYSTEMD=0",
+]
+hostmakedepends = ["gmake"]
+depends = ["perl", "ethtool", "hdparm"]
+pkgdesc = "Optimize Linux Laptop Battery Life"
```

Comment:
Please Don't Randomly Title Case Things

Reply:
Oh I'd just copied that verbatim from https://linrunner.de/tlp/ 😂

Will fix

Reply:
hmm... how about this?

```python
pkgdesc = "Battery life optimization and power management utility"
```

Reply:
something like `Battery life optimization utility` is enough